sunpy/coordinates/_scweb_query.py

```diff
@@ -0,0 +1,80 @@
+"""
+Contains helper functions for generating and sending XML payloads to the SSCWeb API
+(https://sscweb.gsfc.nasa.gov/) to retrieve spacecraft location data.
+"""
+
+import xml.etree.ElementTree as ET
+
+import requests
+
+from astropy.time import Time
+
+from sunpy.time import TimeRange
+
+__all__ = ["_create_xml_request", "_send_requests"]
+
+def _create_xml_request(name, time_range, system):
+    """
+    Create an XML request payload for SSCWeb to fetch location data.
+    As Per the documentation (https://sscweb.gsfc.nasa.gov/WebServices/REST/)
+
+    Parameters
+    ----------
+    name : str
+        Name of the satellite or spacecraft.
+    time_range : sunpy.time.TimeRange
+        The time range for which to fetch location data.
+
+    Returns
+    -------
+    str
+        XML string formatted for SSCWeb API requests.
+    """
+    if not (isinstance(time_range, TimeRange) or isinstance(time_range, Time)):
+        raise ValueError("time_range must be a SunPy TimeRange or astorpy time object.")
+
+    if isinstance(time_range,Time):
+        time_range = TimeRange(time_range[0],time_range[1])
```

Comment:
```suggestion
    if isinstance(time_range, Time):
        time_range = TimeRange(time_range[0], time_range[1])
```
Is there a need to create the time range?

Reply:
Not really
I thought of using TImerange since it provides some level of input sanitization
for example 
-> Regardless of how input is provide end time will be greater than start time 
-> start time and end time can be accessed easily using `obj.start_time` or `obj.end_time`

Reply:
I will let other reviewers decide on this then.